packages/react-native-web/src/modules/createDOMProps/index.js

```diff
@@ -879,8 +879,11 @@ const createDOMProps = (elementType, props, options) => {
       `props.pointerEvents is deprecated. Use style.pointerEvents`
     );
   }
+  const pointerEventsValue =
+    StyleSheet.flatten(style).pointerEvents || pointerEvents;
```

Comment:
My concern here is that this has a good chance of regressing performance because we're flattening the styles before resolving them. That's probably why the special handling of the pointerEvent prop wasn't ported.

It's been a while since I've looked at the e2e benchmarks but that would be the way to confirm this. Might be a good idea to separate the fix for the pointerEvents behavior from the attempt to support inline styles pointerEvent. The latter is an edge case that can easily be avoided by changes to product code anyway

Reply:
What I meant is that if the docs say that it's fine to use plain JS objects people will follow this instruction anyways. That's why I think it is should be covered. Maybe there should be at least a warning in the console to provide some feedback to the library users that something might not work as expected.

I've removed the code from the PR so it includes only the fix for the incorrect `pointerEvents` propagation